src/encoded/types/donor.py

```diff
@@ -29,7 +29,8 @@ class Donor(Item):
         'documents',
         'documents.award',
         'documents.lab',
-        'documents.submitted_by'
+        'documents.submitted_by',
+        'lab'
```

Comment:
Not a big deal, but when we're expanding lists like this we may as well add a trailing comma.  It results in a cleaner looking diff.  Don't resubmit a PR for it, just fyi for the future.
```
    ...
    'documents.submitted_by',
    'lab',
]
```

Reply:
Yeah, I’ve seen inconsistent usage of trailing commas in Python, so I’ve been equally inconsistent as a reaction. I’d like to see trailing commas for Python lists as a standard for us.